src/components/tipRecords/TipRecordComponent.vue

```diff
@@ -17,11 +17,11 @@
       <div>
         <a class="tip__url mb-2 text-ellipsis pr-2" :title="tip.url" :href='tip.url' target="_blank">{{tip.url}}</a>
       </div>
-      <div class="tip__footer text-ellipsis">
+      <div class="tip__footer">
         <div class="row">
           <div class="col-lg-9 col-md-12">
-            <span class="tip__amount" @click="retip(tip.id)" title="Send AE to this post">
-              <img src="../../assets/heart.svg"> {{ tip.total_amount }} <span>AE</span>
+            <span class="tip__amount position-relative" >
+              <span class="tip__amount__btn" title="Send AE to this post"><retip-component :tipid="tip.id"></retip-component></span> {{ tip.total_amount }} <span class="ae">AE</span>
```

Comment:
This line seems to be too long, proposing to split it:
```suggestion
              <span class="tip__amount__btn" title="Send AE to this post">
                <retip-component :tipid="tip.id" />
              </span>
              {{ tip.total_amount }} <span class="ae">AE</span>
```